BUILDING_DETAILS.md

```diff
@@ -2,6 +2,20 @@
 
 This document covers details that may be useful to some users.
 
+## Building additional yocto packages
+
+Assuming the package you want to add is already present in one of the layers
```

Comment:
```suggestion
Assuming the package you want to add is already present in one of the layers of the repos
```

Reply:
That would be misleading. It is not always enough for a package to be in one of the layers within the repos configured in bisdn-linux.yaml. The layer has to be configured, too, often with an explicit line. If we want to open that can of worms, we will need to expand the section quite a bit more. I hope the link to the upstream documentation and the examples contained in bisdn-linux.yaml are sufficient without being overwhelming.